### ISSUE TITLE
Test cases for `train_test_split()` function

### ISSUE THREAD

**Issue by Nazliozum** [UBC-MDS/CrossR#14]
**Type constraints:**
- X and y are data frames
- random_state is integer
- shuffle is a boolean
- test_size is a float between 0 and 1

**Other constraints:**
- Shapes of X and y are consistent
- Threshold for the number of observations in X, suggestion --> (number of observations >=5)
